### PR TITLE
✨ Get EXA

### PR DESCRIPTION
**Pull request by sebipap**

### CODE REVIEW

File: components/getEXA/Routes.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+            key={route.routeId}
+            route={route}
+            tags={[
+              ...(fastestRouteId === route.routeId ? (['fastest'] as const) : []),
+              ...(bestReturnRouteId === route.routeId ? (['best return'] as const) : []),
+            ]}
+          />
+        ))
+      ) : (
+        'no routes'
```
Comment (jgalat) [inline components/getEXA/Routes.tsx:33]: missing translation

File: components/getEXA/SelectRoute.tsx [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+                sx={{ fontSize: '32px' }}
+                type="number"
+                disableUnderline
+                placeholder="0.0"
+                value={qtyOut ? formatEther(qtyOut) : '-'}
+              />
+            )}
+            {exaBalance !== undefined ? (
+              <Typography ml="auto" color="grey.400" fontWeight={400} whiteSpace="nowrap">
+                Balance: {formatNumber(formatUnits(exaBalance, 18))}
```
Comment (jgalat) [inline components/getEXA/SelectRoute.tsx:157]: Missing translation

File: contexts/GetEXAContext.tsx [hunk 3]
```diff
@@ -0,0 +1,626 @@
...
+          owner: walletAddress,
+          spender: swapper.address,
+          value,
+          nonce,
+          deadline,
+        },
+      }).then(splitSignature);
+
+      const permit = {
+        // owner: walletAddress,
```
Comment (jgalat) [inline contexts/GetEXAContext.tsx:284]: remove comment

File: pages/get-exa.tsx [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+          href="/governance"
+          target="_blank"
+          rel="noopener noreferrer"
+          style={{
+            textDecoration: 'underline',
+          }}
+        >
+          Governance
+        </Link>
+        {t(`, or joining the community as an EXA holder.`)}
```
Comment (jgalat) [inline pages/get-exa.tsx:32]: You can use a `Trans` component to embed the link inside. Check its usages within the app.

**Review by jgalat — APPROVED** [review]
lgtm! left some minor comments

File: pages/get-exa.tsx (hunk 4 above)
Comment (sebipap) [reply pages/get-exa.tsx:32]: Didn't we have an issue with `i18n:scan` script when using it? didn't it delete the translations

File: contexts/GetEXAContext.tsx [hunk 5]
```diff
@@ -0,0 +1,626 @@
...
+  const [chains, setChains] = useState<ContextValues['chains']>();
+  const [routes, setRoutes] = useState<ContextValues['routes']>();
+  const [destinationCallData, setDestinationCallData] = useState<ContextValues['destinationCallData']>();
+  const [route, setRoute] = useState<ContextValues['route']>();
+  const [tx, setTX] = useState<ContextValues['tx']>();
+  const [txStep, setTXStep] = useState<ContextValues['txStep']>(TXStep.APPROVE);
+  const [activeRoutes, setActiveRoutes] = useState<ContextValues['activeRoutes']>();
+  const [bridgeStatus, setBridgeStatus] = useState<ContextValues['bridgeStatus']>();
+
+  // const { chain: activeChain } = useNetwork();
```
Comment (franm91) [inline contexts/GetEXAContext.tsx:132]: remove comment

File: i18n/es/translation.json [hunk 6]
```diff
@@ -479,5 +479,30 @@
   "There is no voting power in the connected wallet, and no votes have been delegated to you.": "No hay poder de voto en la billetera conectada, y no se te han delegado votos.",
   "Delegated to you": "Delegado a ti",
   "On connected wallet": "En billetera conectada",
-  "Delegated to": "Delegado a"
+  "Delegated to": "Delegado a",
+  "Transaction Summary": "Resumen de la Transacción",
+  "Estimated Output": "Salida Estimada",
```
Comment (franm91) [inline i18n/es/translation.json:484]: what do you think of "Retiro estimado"?

File: components/getEXA/Route.tsx [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+                display="flex"
+                alignItems="center"
+                height={16}
+                py={0.375}
+                px={0.75}
+                borderRadius={0.5}
+                bgcolor={tag === 'fastest' ? '#33CC59' : '#0095FF'}
+              >
+                <Typography variant="chip" color="components.bg">
+                  {t(tag.toUpperCase())}
```
[the remark this answers is not in the document]
Comment (sebipap) [reply components/getEXA/Route.tsx:69]: you're right, I replaced it with 
```
{tag === 'fastest' ? t('FASTEST') : t('BEST RETURN')}
```

**Review by franm91 — DISMISSED** [review]

File: i18n/es/translation.json (hunk 6 above)
Comment (sebipap) [reply i18n/es/translation.json:484]: I'll use 'Monto' because we're talking about how much EXA you're going to get

**Review by jgalat — APPROVED** [review]

**Review by JuampiRombola — APPROVED** [review]
🚀

**Review by jgalat — APPROVED** [review]

**Pull request merged**